lib/matplotlib/tri/triinterpolate.py

```diff
@@ -241,13 +241,13 @@ class LinearTriInterpolator(TriInterpolator):
 
     Parameters
     ----------
-    triangulation : `~matplotlib.tri.Triangulation`
+    triangulation : `.Triangulation`
```

Comment:
I think we had that topic in another PR lately. This change is against our documentation guide https://matplotlib.org/devel/documenting_mpl.html#referencing-types
> Use full references `~matplotlib.colors.Normalize` with an abbreviation tilde in parameter types. While the full name helps the reader of plain text docstrings, the HTML does not need to show the full name as it links to it. Hence, the ~-shortening keeps it more readable.
>
> Use abbreviated links `.Normalize` in the text.

I still think this is a reasonable policy, but we can discuss that if you like. However, I don't want us do act differently from our stated policy.

Reply:
let's discuss this separately, I reverted the changes for now.